app/src/src/components/common/DropdownList.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { View, StyleSheet, ScrollView, Platform } from 'react-native';
+import { RadioButton, List } from 'react-native-paper';
+
+// a button with a dropdown selector
+const DropdownList = ({ choices, text, set, currChoice, currVal, setZero }) => {
+  const [expand, setExpand] = useState(false);
+  const [value, setValue] = useState(currVal);
+  const press = () => setExpand(!expand);
+  const chooseMsg = 'Choose ' + text;
+  return (
+    <View style={styles.viewStyle}>
+      <List.Accordion
+        style={expand ? styles.accordionOnExpand : styles.accordion}
+        title={expand ? chooseMsg : value ? choices[value] : currChoice}
+        expanded={expand}
+        onPress={press}
+        titleStyle={{ ...styles.title, color: expand ? '#1A5BBB' : '#000' }}
+      >
+        <View style={styles.radioGroup}>
+          <RadioButton.Group
+            style={styles.radioBtn}
+            onValueChange={(value) => {
+              setZero(value === 0 ? 0 : 999);
+              setValue(value);
+              set(value);
+            }}
+            value={value}
+          >
+            <ScrollView style={styles.scrollView} nestedScrollEnabled={true}>
+              {choices.map((choice, i) => (
+                <View
+                  style={i === choices.length - 1 ? '' : styles.radioBtn}
```

Comment:
Maybe instead of '' we use {}? I think style usually is an object so an empty object seems more appropriate.